Library/Homebrew/tap.rb

```diff
@@ -988,19 +988,17 @@ def self.tap_migration_oldnames(current_tap, name_or_token)
   # Array with autobump names
   sig { returns(T::Array[String]) }
   def autobump
-    # TODO: uncomment when official taps are prepared to use new autobump system
-    #
-    # autobump_packages = if core_cask_tap?
-    #   Homebrew::API::Cask.all_casks
-    # elsif core_tap?
-    #   Homebrew::API::Formula.all_formulae
-    # else
-    #   {}
-    # end
-    #
-    # @autobump ||= autobump_packages.select do |_, p|
-    #   p["autobump"] == true && !p["skip_livecheck"] && !(p["deprecated"] || p["disabled"])
-    # end.keys
+    autobump_packages = if core_cask_tap?
+      Homebrew::API::Cask.all_casks
+    elsif core_tap?
+      Homebrew::API::Formula.all_formulae
+    else
+      {}
+    end
+
+    @autobump ||= autobump_packages.select do |_, p|
+      p["autobump"] == true && !p["skip_livecheck"] && !(p["deprecated"] || p["disabled"])
+    end.keys
```

Comment:
This is very hard to follow.

```suggestion
    @autobump ||= autobump_packages.select do |_, package|
      next if p["deprecated"] || p["disabled"]
      next if p["skip_livecheck"]

      package["autobump"] == true
    end.keys
```

assuming that works as expected (please test it yourself). if it doesn't work: please follow this style